azure_arc_k8s_jumpstart/docs/local_microk8s_gitops_helm.md

```diff
@@ -0,0 +1,221 @@
+
+    ![](../img/local_microk8s_gitops_helm/02.png)
+
+* [Install or update Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). **Azure CLI should be running version 2.7** or later. Use ```az --version``` to check your current installed version.
+
+* Export MicroK8s config
+
+    You can export MicroK8s cluster config to a file and use `kubectl` directly instead of `microk8s kubectl` command.
+
+    * Windows
+
+        ```terminal
+        microk8s config view > %HOMEPATH%\.kube\microk8s
+        ```
+    
+    * Linux
+
+        ```terminal
+        microk8s config view > ~\.kube\microk8s
+        ```
+    From this point forward, this guide assumes you have exported MicroK8s config file and have set it in kubectl using `--kubeconfig` flag or `$KUBECONFIG` environment variable. More information can be found in [Organizing Cluster Access Using kubeconfig Files](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/).
+
+# Manually setting up an ingress contoller on MicroK8s
+
+The demo application that will be deployed later in this guide relies on an ingress controller. Given that MicroK8s needs [Multipass](https://multipass.run/) on Windows and MacOS, we'll be covering that scenario and assuming Multipass is running.
+
+## NGINX Controller Deployment
+* Run the following command to install the nginx ingress controller on MicroK8s:
+    ```shell
+    kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v0.35.0/deploy/static/provider/baremetal/deploy.yaml
+    ```
+* This command will create a new namespace and deploy the required components in this namespace. To verify the deployment of the ingress controller was successful, make sure the pod with name `ingress-nginx-controller-<random id>-<random id>` is in a running state with 1/1 containers ready and that a service has been exposed as NodePort.
+
+    ```shell
+    kubectl get pods -n ingress-nginx
+    ```
+    ![](../img/local_microk8s_gitops_helm/03.png)
+
+    ```shell
+    kubectl get svc -n ingress-nginx
+    ```
+
+    ![](../img/local_microk8s_gitops_helm/04.png)
+
+* Take note of the port where the ingress has been exposed (in the image above it was assigned port **32106**). We now need to get the IP address assigned to our microk8s-vm instance in multipass:
+
+    ```shell
+    multipass list
+    ```
+
+    ![](../img/local_microk8s_gitops_helm/05.png)
+
+* Combining the IP address from multipass and the NodePort assigned to the ingress controller, we can now test that the NGINX ingress controller has been deployed successfully. In our case, the full address becomes http://172.22.206.155:32106, so if you open that in a browser or command line, you should get a HTTP 404 reponse with a nginx footer. This shows that the ingress is working. The 404 response is to be expected since you haven't setup an ingress route yet. You will do that in the next section.
```

Comment:
```suggestion
* Combining the IP address from multipass and the NodePort assigned to the ingress controller, we can now test that the NGINX ingress controller has been deployed successfully. In our case, the full address becomes *`http://172.22.206.155:32106`*. 

Using the below in your browser or command line, should get you with a HTTP 404 response with a nginx footer. This shows that the ingress is working. The 404 response is to be expected since you haven't setup an ingress route yet. You will do that in the next section.
```